### DISCUSSION TITLE
How to use this library for .NET Core WebApis

### DISCUSSION THREAD

**Discussion by shripadbhargav** [dotnet/BenchmarkDotNet#2357] [Q&A]
Hello, I am wondering how to use this library for .NET Core Web Api. I have tried but not successfull.

**Comment by adamsitnik** [comment]
Hello @shripadbhargav 

It's not supported by design. You can read more at https://github.com/dotnet/BenchmarkDotNet/issues/1296#issuecomment-715501310

If https://github.com/aspnet/Benchmarks/ is too heavy for you, please consider using bombardier tool.

@davidfowl has recently demoed it: https://youtu.be/1eHjemGKfRw?t=433
